src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Startup.cs

```diff
@@ -27,13 +35,20 @@ public override void ConfigureServices(IServiceCollection services)
 
         public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IServiceProvider serviceProvider)
         {
+#if DEBUG
```

Comment:
Why not use IHostingEnvironment instead?

Reply:
Good point!!! Forgot about that interface. I will make the change tomorrow.

Reply:
Use the value of `IsDevelopment` directly in the default value argument.

Reply:
Sweet :)